src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/integration/endtoend/TierTest.kt

```diff
@@ -33,22 +32,26 @@ class TierTest : MockedEndpointsTestBase() {
 
   @Test
   fun `leaves message on queue if tier calculation cannot be found`() {
-    setupNotFoundTierCalculationResponse()
+    var notFoundRequest = setupNotFoundTierCalculationResponse()
     awsSqsClient.sendMessage(queue, tierUpdateMessage())
-    // the message goes back on the queue but is not visible until after the test ends
-    await untilCallTo { getNumberOfMessagesCurrentlyNotVisibleOnQueue() } matches { it == 1 }
+    Thread.sleep(1000L)
```

Comment:
shame
need to find a better way of working out when the message has been consumed and processed